nvflare/app_common/response_processors/set_global_weights.py

```diff
@@ -0,0 +1,136 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvflare.apis.client import Client
+from nvflare.apis.dxo import DataKind, from_shareable
+from nvflare.apis.fl_context import FLContext
+from nvflare.apis.shareable import Shareable
+from nvflare.app_common.abstract.model import make_model_learnable
+from nvflare.app_common.abstract.response_processor import ResponseProcessor
+from nvflare.app_common.app_constant import AppConstants
+
+
+class WeightMethod(object):
+
+    FIRST = "first"
+    CLIENT = "client"
+
+
+class SetGlobalWeights(ResponseProcessor):
```

Comment:
This is cosmetic. But "SetGlobalWeights" is not a proper class name. It's better changed to "GlobalWeightsProcessor" or "GlobalWeightsInitializer".

Reply:
Thank you. Changed to `GlobalWeightsInitializer`.